src/index.jsx

```diff
@@ -1,2 +1,4 @@
 export { default } from './core/SequenceExamWrapper';
 export { default as OuterExamTimer } from './core/OuterExamTimer';
+export { default as checkExamAccessToken } from './api';
+export { store } from './data';
```

Comment:
That makes sense. Yes, I could use the checkExamAccessToken to fetch/return the access token response. The reason I had exposed the library's store here was in order to check whether this Unit was part of the exam, so I would only make the call to get the access token if it was an exam. So that would maybe be the tradeoff of not exposing the store. Since the frontend-app-learning store doesn't include the exam state I believe. So I think these are the options:

- check exam state from unit before making call to get exam access token (would require store from lib)
- check if exam by looking at content/course id. This would require adding the content id (aka sequence id) as a prop in the unit and then calling to exams to check if it corresponds to an exam. (this call could be another api function from frontend lib special exams maybe)
- always call checkExamAccessToken from the Unit and have this api function return the token if there is one.

Reply:
even if we're always calling `checkExamAccessToken` from the unit, `checkExamAccessToken` can decide if it's an exam and weather or not to make the API call correct?

Reply:
also I think I was a bit hasty with suggesting checkAccessToken both check AND return since that probably it may not be a very react friendly way to do things. I think that depends on how to best deal with making for the xblock doesn't render until that info is available.